sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Vendoring/AzureEventSourceLogForwarder.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Diagnostics.Tracing;
+using Azure.Core.Diagnostics;
+using Azure.Core.Shared;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Monitor.OpenTelemetry.AspNetCore
+{
+    internal sealed class AzureEventSourceLogForwarder : IHostedService
```

Comment:
Can you please move this class out of the Vendoring directory?

The Vendoring dir is used for the code that we copied from OTel. Long term, this directory should be deleted when all the OTel libraries are stable.

I'd suggest making a new directory under "Internals". Something like "Internals\AzureCompat".